pkg/opssight/creater_test.go

```diff
@@ -396,8 +396,8 @@ func validateReplicationControllers(t *testing.T, replicationControllers []*comp
 	}
 
 	for _, rc := range replicationControllers {
-		if !cmp.Equal(rc.GetObj(), expectedReplicationController[rc.GetName()]) {
-			t.Errorf("replication controller is not equal for %s. Diff: %+v", rc.GetName(), cmp.Diff(rc.GetObj(), expectedReplicationController[rc.GetName()]))
+		if !cmp.Equal(rc.ReplicationController, expectedReplicationController[rc.GetName()]) {
```

Comment:
Since this test case functions are commented, it is not called now. I will create a separate PR to uncomment those functions and test the OpsSight orchestration